input/examples/Bundle-multiple-summ-report.json

```diff
@@ -521,7 +521,7 @@
             }
           ]
         },
-        "measure": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-mrp-example",
+        "measure": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-mrp-example|4.0.0",
```

Comment:
The measure example in the IG has has version 1.1.1
    "url": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-mrp-example",
    "version": "1.1.1",